src/Nethermind/Nethermind.AccountAbstraction/Executor/IUserOperationTxBuilder.cs

```diff
@@ -30,7 +30,7 @@ public interface IUserOperationTxBuilder
             byte[] callData,
             Address sender,
             BlockHeader parent,
-            IReleaseSpec spec,
+            IReleaseSpec specFor1559,
```

Comment:
why the changes here?

Reply:
Will address this in the next PR with slotTime in the config.